packages/expo-av/ios/EXAV/Video/EXVideoManager.m

```diff
@@ -28,7 +29,8 @@ - (void)setModuleRegistry:(EXModuleRegistry *)moduleRegistry
 
 - (UIView *)view
 {
-  return [[EXVideoView alloc] initWithModuleRegistry:_moduleRegistry];
+  id<EXAVInterface> avModule = [_moduleRegistry getModuleImplementingProtocol:@protocol(EXAVInterface)];
+  return [[EXVideoView alloc] initWithAvModule:avModule];
 }
 
 - (NSDictionary *)constantsToExport
```

Comment:
do we still need the `EXVideoManager` as we now have the new `VideoViewModule`?

Reply:
We don't, but I thought I could keep both implementations for a moment so that I can easily switch between them when testing Fabric 😅